webapp/src/containers/modals/TaskModal/index.tsx

```diff
@@ -62,6 +65,42 @@ const TaskModal = () => {
         };
     };
 
+    const getAreaPathValuesRequest = useMemo<GetSubscriptionFiltersRequest>(() => ({
+        organization: formFields.organization as string,
+        projectId: selectedProjectId,
+        filters: subscriptionFiltersForBoards,
+        eventType: boardEventTypeOptions[0].value, // We need an eventType field for this API. Since we do not have one in create task modal we are hardcoding it.
```

Comment:
```suggestion
        eventType: boardEventTypeOptions[0].value, // We need an eventType field for this API. Since we do not have one in the create task modal so, we are hardcoding it.
```